features/step_definitions/purchase_steps.rb

```diff
@@ -0,0 +1,30 @@
+Given(/^there are some books in the database$/) do
+  FactoryGirl.create_list(:book, 100)
+end
+
+When(/^I click the "(.*?)" button for the first book$/) do |button|
```

Comment:
Unused block argument - `button`. You can omit the argument if you don't care about it.